doc/stake-pool-operations/KES_period.md

```diff
@@ -25,21 +25,22 @@ We check the current tip of the blockchain:
     cardano-cli query tip --mainnet
 
     {
-        "blockNo": 36914,
-        "headerHash": "58df595137e71c0fa65edc99add11704b00e5f163475bd804e4bd59c126bfc9b",
-        "slotNo": 8520857
+        "epoch": 259,
+        "hash": "dbf5104ab91a7a0b405353ad31760b52b2703098ec17185bdd7ff1800bb61aca",
+        "slot": 26633911,
+        "block": 5580350
     }
 
-In this example, we are currently in slot 8520857, and we know from the genesis file that one period lasts for 129600 slots. So we calculate the current period by
+In this example, we are currently in slot 26633911, and we know from the genesis file that one period lasts for 129600 slots. So we calculate the current period by
 
-    expr 8520857 / 129600
-    > 65
+    expr 26633911 / 129600
```

Comment:
Is this calculation valid? For a pure Shelley testnet it would be but for mainnet?